src/sys/mod.rs

```diff
@@ -143,6 +143,10 @@ feature! {
 #[allow(missing_docs)]
 pub mod sysinfo;
 
+#[allow(missing_docs)]
```

Comment:
I would appreciate it if these new APIs have nice documents or even better, examples:)

Reply:
Fixed, removed the allow lint and added some doctest, please tell me if anything isn't clear in it.